HelpSource/Classes/Buffer.schelp

```diff
@@ -66,7 +66,7 @@ The number of channels for the Buffer. The default is 1.
 argument:: completionMessage
 A valid OSC message or a Function which will return one. A Function will be passed this Buffer as an argument when evaluated.
 argument:: bufnum
-An explicitly specified buffer number. Generally this is not needed.
+An explicitly specified buffer number. Use of this argument is unnecessary and highly discouraged. The bufnum specified here will not be reserved, and may get allocated elsewhere in the future with unintended results. The best practice is to leave this unset and let the buffer number allocator choose the next available bufnum.
```

Comment:
a few things:
1. "unnecessary" - this implies it could be deprecated. I think it is more accurate to leave it more like the original and say "use of this argument is discouraged and unnecessary under normal usage."

2. Could/should this same message or a variant also go on similar methods in this same document? Most class methods on Buffer seem to have similar contracts: `allocConsecutive`, `read`, `readChannel`,  `readNoUpdate`, etc. I don't know for sure though. @jamshark70 @muellmusik ?

3. As the original submitter of the complaint about wording here, I think the way it's written would still have confused me the first time. Can I suggest something? After the "discouraged" sentence:

"Providing a non-nil argument here will bypass the server's buffer allocator. Therefore, the buffer returned by this method may be allocated elsewhere in the future with unintended results."

4. Thank you for this, I really appreciate it. :)

Reply:
> Could/should this same message or a variant also go on similar methods in this same document? Most class methods on Buffer seem to have similar contracts: allocConsecutive, read, readChannel, readNoUpdate, etc. I don't know for sure though. @jamshark70 @muellmusik ?

Um I think @jamshark70  is correct about documenting usage, and I think we don't want to confuse new users with too much info that probably isn't relevant. A 'probably not for you' message can be good, we just want the detail available. So I'd suggest something like 'under normal usage this is not needed and requires special handling' and then link to a section discussing exotic cases and how the allocators etc. work. Do this once and link from relevant methods.

The same should probably be done for Synth:basicNew and the like. Could all be discussed in one place perhaps.